circleguard/widgets.py

```diff
@@ -12,6 +13,7 @@
 
 
 SPACER = QSpacerItem(100, 0, QSizePolicy.Maximum, QSizePolicy.Minimum)
+MAX = 128
```

Comment:
what the HECK is the point of this max constant? why is it 128?

Reply:
`MAX = 128` is just a big number so the checkbox/combobox is on the right.
`setLayoutDirection(Qt.RightToLeft)` can't be used since the text would also be aligned on the right side. 
Open for better solutions